DISPATCH_SUBSCRIBE.md

```diff
@@ -0,0 +1,98 @@
+# Forensics on RDX usage
+
+This is a "working" guess on how RDX works, and how to implement it into your own system.
+
+- working - because it's enough code to actually prove that it functions as advertised
+- guess - in that "Who knows if this is the right way, or not"
+
+## What is the source of this document?
+
+This document is taken entirely from examining the code from the March 2020 version of rdx-demo, and trying to replicate what that does within the firebase sign out code
+
+## What is the high level objective?
+
+- one WC would be able to pass a value into the rdx state model
```

Comment:
This is the wrong terminology and possibly confusing. You dispatch an action, as per redux, but rdx makes it easier by providing the actionCreators for you based on the reducers